utils/utils.py

```diff
@@ -21,29 +21,43 @@ def preprocess_data(X, y=None, remove_zerovar=True, standardize=True):
     return X, y
 
 
-def prox_l1sorted(v, w):
-    """Proximal operator of the OWL norm dot(w, reversed(sort(v)))
+def prox_l1sorted(beta, lambdas):
```

Comment:
I still think we should change the names of the two prox implementations.

Reply:
Yep, we discussed it with Amélie and she'll finish the PR this week :)

Reply:
Actually [I already changed the names](https://github.com/benchopt/benchmark_slope/pull/4/commits/441d4fe2e3cec143b432268f177ff232ec8511f1). And yes I will finish this as soon as I can, thank you for the review